tests/test.rs

```diff
@@ -111,9 +114,12 @@ fn test_all_prop(mut string: String, commands: Vec<Command>) -> bool {
             return false;
         }
     }
-    return true;
+
+    true
 }
 
+// Can we remove random tests? quickcheck-based tessts are probably more
```

Comment:
Maybe yes, maybe no, I'm confused by the fact that quickcheck tests pass very quickly, which means the size of the input is not large